### DISCUSSION TITLE
Roadmap ?

### DISCUSSION THREAD

**Discussion by dingedi** [argosopentech/argos-translate#184] [General]
Is there a roadmap or a list of important things to do on argostranslate?

**Comment by PJ-Finlay** [comment]
Thanks for the interest!

I'm currently mostly working on the training scripts to better automate training and train more and better models. There are also a number of open tickets for varipus smaller things.

Looking forward, breaking changes in 2.0 are still a ways off but I want to do [single character tokenization](https://forum.opennmt.net/t/single-character-tokenization/4464/) and [seq2seq sentence boundary detection](https://youtu.be/dCe2d6oPLZA). Depending on how the field progresses few shot translation may also play a larger role in later versions, but is already implemented.
